src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
If I remember correctly this is a compilation issue with ```  y(luma: T)[alpha: T] => new;``` with the error ```trait not implemented for <D65, T>```. It is fixed by changing it to use the white point